src/aap_eda/wsapi/consumers.py

```diff
@@ -217,6 +217,18 @@ def insert_audit_rule_data(self, message: ActionMessage) -> None:
             )
 
             logger.info(f"Audit rule [{audit_rule.name}] is created.")
+
+            activation_instance = models.ActivationInstance.objects.get(
+                pk=message.activation_id
+            )
+            activation = models.Activation.objects.get(
+                pk=activation_instance.activation_id
+            )
+            if message.ruleset in activation.ruleset_stats:
+                activation.ruleset_stats[message.ruleset][
+                    "rulesTriggered"
```

Comment:
@Dostonbek1 This is controlled from the ansible-rulebook side we cannot increment this on the server side. A session stat is sent at every heartbeat interval, if no events have arrived into the ansible-rulebook we will still be sending the data from the ansible-rulebook and this code will incorrectly say rules have been triggered when no events have been processed. This seems like a different issue related to https://issues.redhat.com/browse/AAP-13874